src/test/fixtures/strict_typing/return_statement_4.fixture.php

```diff
@@ -33,15 +33,14 @@ class MorePointEx extends MorePoint {
 
 }
 
-// this seems failing since 2021.1, it's a PhpStorm bug
-/** @return MorePointEx */
+/** @return Point */
```

Comment:
As I remember, the purpose of this test was to return a derived class, not a base class